addon/-private/class/modifier.ts

```diff
@@ -16,7 +16,26 @@ function deprecateForDestroyables<S>(
   instance: ClassBasedModifier<S>
 ): void {
   deprecate(
-    `Modifier.${name} is deprecated`,
+    `Modifier.${name} is deprecated. Use the corresponding API from '@ember/destroyable' instead.`,
```

Comment:
Total bike shedding here, but when I see `Modifier.foo` I think you mean a static class method (not a prototype)

Reply:
Same basic issue as I mentioned in https://github.com/ember-modifier/ember-modifier/pull/221#discussion_r833596427, we need to include information about the instance so folks can actually figure out which modifiers are triggering the deprecation

Reply:
Fixed both of these issues (and it's good bike shedding!).